### ISSUE TITLE
[v13.0.0-beta.0] Missing keyboard shortcuts

### ISSUE THREAD

**Issue by microbit-robert** [google/blockly#9703]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Description

The following keyboard shortcuts are currently missing if the aim is parity with the keyboard navigation plugin:

- Duplicate (d)
- Clean up workspace (c)
- 'list_shortcuts' is never installed and [this function will always return early](https://github.com/RaspberryPiFoundation/blockly/blob/v13/packages/blockly/core/hints.ts#L76-L77)
    -  I am also not sure how this hint is triggered, I've not managed to get into t[his branch](https://github.com/RaspberryPiFoundation/blockly/blob/v13/packages/blockly/core/block_svg.ts#L1932) when testing

### Reproduction steps

1. 
2. 
3.

### Stack trace

```shell

```

**Comment by maribethb** [comment]
The list shortcuts action is not going into core, by design, so that's going to take a little bit longer before we have a separate standalone plugin for the modal for the list of shortcuts. Makecode uses their own help window though, right? if you name that shortcut `list_shortcuts` then it should work? I'm going to file a separate tracking issue for the plugin for the list shortcuts modal. 

There are not very many blocks that don't have an "in" node so it's more rare to see that message than the arrow key one, but one example is the "random fraction" math block.

**Comment by maribethb** [comment]
Fixed by #9728 and #9727 

Filed https://github.com/RaspberryPiFoundation/blockly-samples/issues/2687 for the list_shortcuts

**Comment by microbit-robert** [comment]
@maribethb Yes, ok, I can now trigger help hint following the changes in https://github.com/RaspberryPiFoundation/blockly/pull/9725.

However, there is no function exposed to clear the help hint, and the help hint id is also not exposed. So embedding applications have to do something like this at the moment to hide the help hint when the 'list_shortcuts' shortcut is triggered:
```
callback: (workspace) => {
    Blockly.Toast.hide(workspace, "helpHint");
```

"list_shortcuts" is also not the [shortcut names enum],(https://github.com/RaspberryPiFoundation/blockly/blob/v13/packages/blockly/core/shortcut_items.ts#L37-L67) should it be? It also has to be a hardcoded string at the moment.

**Comment by maribethb** [comment]
I am working on a design for the plugin that will address these kinds of things in more detail - how the plugin will interact with core and the toasts. 

Would you prefer in the meantime that we just remove this hint until that design is fully baked? Or is the current situation ok for beta testing and you're just making sure we're aware of the issues for the future?

**Comment by microbit-robert** [comment]
Sounds good. Current situation will be fine for beta testing. Yes, it's more about ease of use for other Blockly consuming apps for the release.
